.github/workflows/author_approval.yml

```
@@ -29,7 +29,7 @@ jobs:
           COMMENTER: ${{ github.event.comment.user.login }}
         shell: julia --compile=min --optimize=0 --color=yes {0}
         run: |
-          m = match(r"Created by: @([A-Za-z0-9]*+)(?:$|\n)", ENV["PR_BODY"])
+          m = match(r"Created by: @([^\s]+)", ENV["PR_BODY"])
```

We don't need the `^`, do we?
```suggestion
          m = match(r"Created by: @([\s]+)", ENV["PR_BODY"])
```

---

Disregard previous comment.